ml-agents.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
```

Comment:
Is there a reason you didn't use the canonical filename `Dockerfile`?

Reply:
there can be multiple Dockerfiles, e.g. a cpu version and a gpu version so I prefer to use names such as `ml-agents-cpu.Dockerfile` and `ml-agents-gpu.Dockerfile`. It improves readability in my opinion.